troposphere/static/js/components/Header.react.js

```diff
@@ -158,8 +161,21 @@ let Header = React.createClass({
         this.setState({windowWidth: window.innerWidth});
     },
 
+    handleNotice: function() {
+        if (context.hasMaintenanceNotice()) {
+            NotificationController.warning(
+                "Atmosphere Maintenance Information",
```

Comment:
We may want to generalize this string so that it can be used in "An informational context" outside of a maintenance.